hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestFileTruncate.java

```diff
@@ -671,10 +676,10 @@ public void testTruncateFailure() throws IOException {
           "File does not exist", expected);
     }
 
-    
+
     fs.setPermission(p, FsPermission.createImmutable((short)0664));
     {
-      final UserGroupInformation fooUgi = 
```

Comment:
No changes here include L758,L762,L814,L867 etc? Just suggest to keep it if no explicit changes.